srt-tokio/src/builder.rs

```diff
@@ -81,11 +90,12 @@ pub enum ConnInitMethod {
 }
 
 impl SrtSocketBuilder {
-    /// Defaults to binding to `0.0.0.0:0` (all adaptors, OS assigned port), 50ms latency, and no encryption.
+    /// Defaults to binding to all adaptors, OS assigned port, 50ms latency, and no encryption.
```

Comment:
Add a note that with listen mode, it defaults to IPV4, and in the docs for the `listen` function